### DISCUSSION TITLE
How to Delete a Specific Project or Tracked File

### DISCUSSION THREAD

**Discussion by khanra17** [muety/wakapi#672] [Q&A] [answered]
Sometimes, unrelated activities get tracked, including when I'm not working, and they usually appear under the "unknown project." In Wakatime, I can delete a specific file or an entire project directly on the website. How can I do the same in Wakapi?

**Comment by muety** [comment, marked as answer]
Hi @khanra17, unfortunately, this is not currently supported by Wakapi. #440 is related, I'll re-prioritize it in view of your feature request.

Temporary workaround: if you're using Wakapi.dev, you may send me a message with the data you'd want to be deleted. If you're running your own instance, you can delete the data using raw SQL on the database.

**Comment by khanra17** [comment]
That's the most important feature in a time tracking software according to me.
Its inaccurate & preety much unusable without that.
I will switch back to Wakatime 👋

**Reply by muety** [reply]
I'd argue the most important feature in time tracking software is time tracking. But yeah, in this case you're probably better off with Wakatime 👋.

**Reply by khanra17** [reply]
I was actually listing it as the second most important thing, but if an app can't even track time, can it really be called a time tracker? That's kind of a given, right? 🤡 Anyway, I'm not here to play word games—you get my point! I can track time with a clock if I wanted to, but I'm using automated software for a reason. If it's inaccurate, then it's pretty much useless!
